azure-pipelines.yml

```diff
@@ -0,0 +1,70 @@
+name: 3.0.0-ci.$(Date:yyyyMMdd).$(Rev:r) # equal to $(Build.BuildNumber)
```

Comment:
Is it possible to keep the version number as variable?

Reply:
IIRC `3.0.0-ci.$(Date:yyyyMMdd).$(Rev:r)` should be available during each job via `$(Build.BuildNumber)`. But (again IIRC), you can't assign/access normal variables at name-definition time.

Reply:
This seems to mean that for now our config only produces CI packages - I don't see any logic for creating release packages, prerelease packages, etc. etc.

We can always add these later though, after this PR is merged.